tests/step_implementers/package/test_maven_package.py

```diff
@@ -438,60 +438,6 @@ def test_mvn_quickstart_single_jar_java_error(self, mvn_mock):
                     'Error invoking mvn:.*'):
                 factory.run_step('package')
     
-    @patch('sh.mvn', create=True)
```

Comment:
Why are you striking these?

Reply:
because they were testing that the arifctId and groupId were in the pom.xml when that step doesnt' even use them.